### DISCUSSION TITLE
imported as 'usePrepareContractWrite' was not found in 'wagmi'

### DISCUSSION THREAD

**Discussion by github-Dheeraj** [wevm/wagmi#766] [Q&A]
usePrepareContractWrite is not found in WAGMi is there any other dependency fro this hook to be used .

**Comment by tmm** [comment]
What version are you using? `usePrepareContractWrite` is new in 0.6.

**Reply by github-Dheeraj** [reply]
I update it to 0.6.0 
Now I have this issue


ERROR in ./src/App.jsx 48:16-33
export 'InjectedConnector' (imported as 'InjectedConnector') was not found in 'wagmi/connectors/injected' (module has no exports)
